src/server/graphql/types/User.js

```diff
@@ -103,6 +107,12 @@ const User = new GraphQLObjectType({
       type: GraphQLString,
       description: 'The application-specific name, defaults to nickname'
     },
+    // presence: {
```

Comment:
Code comment reminder

Reply:
removed in a subsequent PR (don't wanna cause too many conflicts)